homepage.php

```diff
@@ -52,57 +52,57 @@
 				<a href = "Query8.php"><img src = "images/Indie.jpg" class="image1"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query8.php">Indie</a></div>
+    				<div class="text"><a href = "Query8_admin.php">Indie</a></div>
   				</div>
 			</div>
 			<div class="images2">
 				<a href = "Query7.php"><img src = "images/R&B.jpg" class="image2"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query7.php">R and B</a></div>
+    				<div class="text"><a href = "Query7_admin.php">R and B</a></div>
```

Comment:
2/9/2021: So I make a page called Query7_admin will mostly the same with Query7 but the only difference was in Query7_admin I use the admin nav which can access to the setting. So for admin page I should make it to link to Query7_admin.